extensions/sql-database-projects/src/extension.ts

```diff
@@ -15,7 +15,9 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 	controllers.push(mainController);
 	context.subscriptions.push(mainController);
 
+	console.log(`Sql-database-projects activate start time:${new Date().getTime()}`);
```

Comment:
if possible, I think we should add this type of logging in test rather than product.

Reply:
I figure at some point we'll just do a full scrub of console.log statements to clean this (and other leftover print-debugging) stuff up.  Maybe `console.debug()` could be used instead?

Reply:
I will get rid of these logs, since they aren't valuable anyways.